editor/src/components/canvas/dom-walker.ts

```diff
@@ -893,6 +893,13 @@ function getSpecialMeasurements(
     height: globalFrame.height - border.top - border.bottom,
   })
 
+  // TODO is there a nicer way to get the number only if it's in pixels, and otherwise set it to null?
```

Comment:
You could `flatMapEither` it and then `defaultEither` it, but it's arguable which is more comprehensible.